.editorconfig

```diff
@@ -0,0 +1,13 @@
+# editorconfig.org
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.md]
+trim_trailing_whitespace = false
```

Comment:
Waar wordt de editorconfig voor gebruikt ? is dat voor visual studio code?

Reply:
Is een file om ervoor te zorgen dat je project layout (qua charset, wat er gebeurt bij een end of line enz.) in elke editor hetzelfde blijft. (dus niet afhankelijk is van de editor die je gebruikt)

Reply:
Ahh oke nice. kende ik niet. en dat werkt out of the box? als in, hoeven we verder niks voor te installeren ?

Reply:
Hiervoor niet idd